linux_sql/README.md

```diff
@@ -1,2 +1,169 @@
-# Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+#Linux Cluster Monitoring Agent
```

Comment:
# Linux Cluster Monitoring Agent
miss a space after the hash

https://github.com/jarviscanada/jarvis_data_eng_OlenaHlotova/blob/42b060b08c394effcf2ff5c5491c2a44a66d7e7c/linux_sql/README.md